packages/salesforcedx-vscode-core/src/commands/deploySourcePath.ts

```diff
@@ -80,7 +80,8 @@ export const deploySourcePaths = async (
       )
     );
 
-    await commandlet.run();
+    const result = await commandlet.run();
+    return result;
```

Comment:
The type of result here is void, which does not look right. You might need to update the type definition in `packages/salesforcedx-vscode-core/src/commands/util/commandletExecutor.ts` and `packages/salesforcedx-vscode-core/src/commands/util/sfCommandlet.ts`

Reply:
Yeah, there's an inconsistency here. However, if I touch commandletExecutor + sfCommandlet, it forces me to uplift the other 20+ commands. Would we be okay leaving a comment in here notifying that currently only 1 command properly returns boolean, and the rest will return undefined?